NuCivic/dkan
# Build DKAN with datastore REST API updates in dataset and datastore modules

Issue: CIVIC-3123

## Description
We need to import automatic to datastore any resource updated via endpoint call.

## User story / stories
- [x] As a DKAN content creator, I want to be able to add resources directly to the datastore, without having to do an extra manual step.

## Acceptance test
- [ ] Do a call via endpoint to update a resource. Use https://github.com/NuCivic/pydkan

## PR dependencies
https://github.com/NuCivic/dkan_dataset/pull/240.
https://github.com/NuCivic/dkan_datastore/pull/70

… #CIVIC-3123

Steps to test:
 1.- clone repo https://github.com/NuCivic/pydkan
 2.- Check if you have the command pip in another case use http://docs.python-guide.org/en/latest/starting/install/osx/
 3.- pip install virtualenv
 4.- Install client as library
  5.- mkdir dkan_api_tests
  6.- cd dkan_api_tests
  7.- virtualenv .
  8.- source bin/activate
  9.- pip install git+git://github.com/NuCivic/pydkan.git@master#egg=pydkan
 10.- (back again to pydkan folder) cd pydkan
 14.- Access to folder examples and edit attach_file_to_node.py 
 15.-  And change the values DKAN_URI = 'http://docker:32782' (your dkan host), RESOURCE_ID = 5 (nid from your local instance), CSV = os.path.join('/Users/teofilosibileau/Downloads', 'CSV.csv') (A csv example on your local)
